### PR TITLE
WIP: Fix rosa cli execute

### PR DESCRIPTION
**Pull request by oharan2**
##### What this PR does / why we need it:
Updated with recent changes required from merged [#16](https://github.com/RedHatQE/rosa-python-client/pull/16)

##### Bug:
Depends and blocked by [#23](https://github.com/RedHatQE/rosa-python-client/issues/23)

### CODE REVIEW

File: tests/conftest.py [hunk 1]
```diff
@@ -64,11 +64,12 @@ def ocp_target_version(request):
 
 
 @pytest.fixture(scope="session")
-def rosa_regions(rosa_allowed_commands):
+def rosa_regions(rosa_allowed_commands, ocm_token):
```
Comment (myakove) [inline tests/conftest.py:67]: do you have a fixture called `ocm_token`?
Also rosa can get ocm client, I think it's better to send it and not the token

File: tests/conftest.py [hunk 2]
```diff
@@ -64,11 +64,12 @@ def ocp_target_version(request):
...
     # A region (any region) is required for ROSA commands
     return rosa.cli.execute(
         command="list regions --region us-west-2",
```
Comment (myakove) [inline tests/conftest.py:70]: `--region us-west-2` this will fail with the new changes.
remove it from the command

**Review by myakove — CHANGES_REQUESTED** [review]

File: tests/conftest.py (hunk 1 above)
Comment (oharan2) [reply tests/conftest.py:67]: I can pass client, but [there is a fixture](https://github.com/RedHatQE/managed-services-integration-tests/blob/main/tests/conftest.py#L51) that pass the `ocm_token`, so we have it anyway.

**Comment by oharan2** [comment]
Resolved in #197  with passing ocm_client rather than ocm_token

**Pull request closed without merging**